### DISCUSSION TITLE
mono repo with yarn workspace, built successfully but only static files is served and not handling 404 back to index.html

### DISCUSSION THREAD

**Discussion by u007** [vercel/community#1438] [Help] [answered]
hi

when i tried to run vercel dev, it works perfectly.
but after deployed, i get only static build working (root client in vuejs) but the sub folder server (node) based did not run.

here is my vercel.json

```vercel.json
{
  "version": 2,
  "installCommand": "yarn install",
  "builds": [
    { "src": "package.json", "use": "@now/static-build", "config": { "distDir": "dist" } },
    { "src": "dist/**/*", "use": "@vercel/static" },
    { "src": "server/dist/**/*.js", "use": "@vercel/node" }
  ],
  "routes": [
    {
      "src": "/api/trpc/(.*)",
      "dest": "server/dist/src/server.js"
    },
    { "handle": "filesystem" },
    {
      "src": "/(.*)",
      "dest": "dist/index.html"
    }
  ]
}

```

i have api proxied from to /api/trpc and it works in dev, im not sure why node is not running

package.json
```package.json
//...
"scripts": {
    "dev": "concurrently \"yarn dev-server\" \"yarn dev-client\"",
    "dev-client": "vite",
    "dev-server": "cd server && yarn dev",
    "test": "ng test",
    "build:server": "cd server && yarn build",
    "build:client": "vite build",
    "build": "yarn build:server && yarn build:client",
  },
```
 i have root directory as vue3 with vite, and sub folder server for expressjs node

**Comment by amyegan** [comment, marked as answer]
Is your Express code set up as a Serverless Function? If not, you should take a look at the guide on [using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel). It covers the setup steps and some [drawbacks](https://vercel.com/guides/using-express-with-vercel#drawbacks-and-edge-cases) to consider when running Express in a serverless environment
